<?xml version="1.0" encoding="UTF-8"?>  
<launch>

    <arg name="velocity_coeff" default="0.055555"/>
    <arg name="radius_tube"    default="250"/>
    
    <node pkg="marker_move" type="go_in_circle" name="go_in_circle" output="screen">

    	<param name="Velocity_coeff" value="$(arg velocity_coeff)"/>
    	<param name="Radius_tube"    value="$(arg radius_tube)"/>
    </node>

  </launch> 
